README.md

```diff
@@ -126,6 +126,42 @@ $ export KUBERNETES_MASTER=http://${servicehost}:8888
 You can increase the verbosity of the logging for the API server, scheduler, and/or the controller-manager by including, for example, `--v=2`.
 This can be very helpful while debugging.
 
+#### Scheduler Configuration
```

Comment:
this is great information to document, but I'd rather see it in some kind of "tuning" document vs. in the main readme.